docs/how_to/create_a_quantum_circuit.rst

```diff
@@ -0,0 +1,169 @@
+
+########################
+Create a quantum circuit
+########################
+
+This guide shows you how to create a quantum circuit.
+
+There are two ways to create a :class:`~.QuantumCircuit` object:
+
+1. Initialize :class:`~.QuantumCircuit` by directly specifying the number of qubits and classical bits you want.
+2. Creating :class:`~.QuantumRegister`\ s and :class:`~.ClassicalRegister`\ s and use the registers to initialize a :class:`~.QuantumCircuit`
+
+.. _create circuit with qubit numbers:
+
+Create by specifying the number of qubits and classical bits
+============================================================
+
+You can create a :class:`~.QuantumCircuit` by only specifying the number of qubits and classical bits. For example:
+
+.. testcode::
+
+    from qiskit import QuantumCircuit
+
+    # Initialize number of qubits and classical bits
+    n_qubits = 3
+    n_bits = 2
+
+    # Create the circuit
+    qc = QuantumCircuit(n_qubits, n_bits)
+    print(qc.draw())
+
+.. testoutput::
+    :options: +NORMALIZE_WHITESPACE
+
+    q_0: 
+        
+    q_1: 
+        
+    q_2: 
+        
+    c: 2/
+     
+
+You can also create a :class:`~.QuantumCircuit` with only qubits and no classical bits, by omitting the number of classical bits:
+
+.. testcode::
+
+    from qiskit import QuantumCircuit
+
+    qc = QuantumCircuit(n_qubits)
+    print(qc.draw())
+
+.. testoutput::
+    :options: +NORMALIZE_WHITESPACE
+
+    q_0: 
+        
+    q_1: 
+        
+    q_2: 
+
+Create from quantum and classical registers
```

Comment:
same as above, just using "create" on it's own like this feels a bit odd grammatically. I think using something like "create a circuit from..." is a bit clearer